src/MySqlConnector/Core/ServerSession.cs

```diff
@@ -159,7 +159,7 @@ public void FinishQuerying()
 				// KILL QUERY will kill a subsequent query if the command it was intended to cancel has already completed.
 				// In order to handle this case, we issue a dummy query that will consume the pending cancellation.
 				// See https://bugs.mysql.com/bug.php?id=45679
-				Log.Info("{0} sending 'DO SLEEP(0)' command to clear pending cancellation", m_logArguments);
+				Log.Info("Session{0} sending 'DO SLEEP(0)' command to clear pending cancellation", m_logArguments[0]);
```

Comment:
This should remain as `m_logArguments` to avoid allocating a `params object[]` array when calling the logging method.